### PR TITLE
homework3

### PR DESCRIPTION
**Pull request by YeaaaaH**

### CODE REVIEW

File: HW/src/main/java/home_work_03/MergeArrays.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package home_work_03;
```
Comment (Siril-O) [inline HW/src/main/java/home_work_03/MergeArrays.java]: https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

Comment (Siril-O) [reply HW/src/main/java/home_work_03/MergeArrays.java]: Package names are all lowercase, with consecutive words simply concatenated together (no underscores)

File: HW/src/main/java/home_work_03/MergeArrays.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    /**
+     * Merging two arrays
+     * @param arr1 array#1
+     * @param arr2 array#2
+     * @return merged array
+     */
+    public static int[] mergeArrays(int[] arr1, int[] arr2) {
+        int newlength = arr1.length + arr2.length;
+        int[] arr = new int[newlength];
+        //Index Position in first array - starting with first element
```
Comment (Siril-O) [inline HW/src/main/java/home_work_03/MergeArrays.java]: just name it firstIndex and you will not need this comment

File: HW/src/main/java/home_work_03/MergeArrays.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+public class MergeArrays {
+    /**
+     * Merging two arrays
+     * @param arr1 array#1
+     * @param arr2 array#2
+     * @return merged array
+     */
+    public static int[] mergeArrays(int[] arr1, int[] arr2) {
+        int newlength = arr1.length + arr2.length;
+        int[] arr = new int[newlength];
```
Comment (Siril-O) [inline HW/src/main/java/home_work_03/MergeArrays.java]: name it newArray or mergedArray  or result

File: HW/src/main/java/home_work_03/TheatricalPlay.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+     * @param roles roles array
+     * @param textLines replicas
+     * @return whole play String
+     */
+    private static String printTextPerRole(final String[] roles, final String[] textLines) {
+        StringBuilder result = new StringBuilder();
+        for (int role = 0; role < roles.length; role++) {
+            result.append(roles[role].concat(":"));
+            result.append("\n");
+            for (int text = 0; text < textLines.length; text++) {
```
Comment (Siril-O) [inline HW/src/main/java/home_work_03/TheatricalPlay.java]: rename text it is not text

Comment (YeaaaaH) [reply HW/src/main/java/home_work_03/TheatricalPlay.java]: done

File: HW/src/main/java/home_work_03/MergeArrays.java (hunk 3 above)
Comment (YeaaaaH) [reply HW/src/main/java/home_work_03/MergeArrays.java]: done

File: HW/src/main/java/home_work_03/MergeArrays.java (hunk 2 above)
Comment (YeaaaaH) [reply HW/src/main/java/home_work_03/MergeArrays.java]: done

File: HW/src/main/java/home_work_03/MergeArrays.java (hunk 1 above)
Comment (YeaaaaH) [reply HW/src/main/java/home_work_03/MergeArrays.java]: done

**Pull request merged**